src/hooks/useSpaceLayoutProps.ts

```diff
@@ -0,0 +1,17 @@
+import { useMemo } from 'react'
+import { space, layout } from 'styled-system'
+import { IndexSignature, pick, reject } from '../utilities'
+/**
+ * Split out styled-system's space/layout props
+ *
+ * @param props - an object of properties
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing JSDoc parameter type for 'props'.](https://app.codacy.com/manual/PrideInLondon/pride-london-web/pullRequest?prid=4765408)